prow/integ-suite-kind.sh

```diff
@@ -79,6 +80,16 @@ while (( "$#" )); do
   esac
 done
 
+# TODO(landow): remove this, just making sure this works before creating a separate Prow job
```

Comment:
I'm assuming we'll want to make a separate Prow job for this - all multicluster tests should succeed in both multinetwork and single network scenarios.